testproject-tools-integration/Assets/Tests/Runtime/SceneEventTests.cs

```diff
@@ -250,9 +250,9 @@ public IEnumerator TestS2CUnloadSent()
             // as this is when the message is sent
             m_ServerNetworkSceneManager.OnSceneEvent += sceneEvent =>
             {
-                if (sceneEvent.SceneEventType.Equals(SceneEventType.LoadComplete) && && sceneEvent.ClientId == Server.LocalClientId)
+                if (sceneEvent.SceneEventType.Equals(SceneEventType.LoadComplete) && sceneEvent.ClientId == Server.LocalClientId)
                 {
-                    serverSceneLoaded = true;
+                    serverSceneUnloaded = true;
```

Comment:
oops, I think the variable should be named `serverSceneLoaded` looking at the condition. 
Change line 248 to `var serverSceneLoaded = false;`